src/java/org/apache/cassandra/schema/SchemaKeyspace.java

```diff
@@ -857,6 +891,36 @@ static Keyspaces fetchNonSystemKeyspaces()
         return fetchKeyspacesWithout(SchemaConstants.LOCAL_SYSTEM_KEYSPACE_NAMES);
     }
 
+    public static void validateNonCompact() throws StartupException
+    {
+        String query = format("SELECT keyspace_name, table_name, flags FROM %s.%s", SchemaConstants.SCHEMA_KEYSPACE_NAME, TABLES);
+
+        StringBuilder messages = new StringBuilder();
+        for (UntypedResultSet.Row row : query(query))
+        {
+            if (SchemaConstants.isLocalSystemKeyspace(row.getString("keyspace_name")))
+                continue;
+
+            Set<String> flags = row.getFrozenSet("flags", UTF8Type.instance);
+            if (TableMetadata.Flag.isLegacyCompactTable(TableMetadata.Flag.fromStringSet(flags)))
+            {
+                messages.append(format("ALTER TABLE %s.%s DROP COMPACT STORAGE;\n",
+                                          maybeQuote(row.getString("keyspace_name")),
+                                          maybeQuote(row.getString("table_name"))));
```

Comment:
It seems this has lost its alignment:
```suggestion
                messages.append(format("ALTER TABLE %s.%s DROP COMPACT STORAGE;\n",
                                       maybeQuote(row.getString("keyspace_name")),
                                       maybeQuote(row.getString("table_name"))));
```